tests/sentry/backup/test_models.py

```diff
@@ -217,6 +228,74 @@ def test_organization(self):
         self.create_organization(owner=user)
         return self.import_export_then_validate()
 
+    @targets_models(Actor)
+    def test_actor(self):
+        self.create_user(email="test@example.com")
+        self.create_team(name="pre save team", organization=self.organization)
+        return self.import_export_then_validate()
+
+    @targets_models(ApiAuthorization)
+    def test_apiAuthorization(self):
+        user = self.create_user()
+        app = ApiApplication.objects.create(name="test", owner=user)
+        ApiAuthorization.objects.create(
+            application=app, user=self.create_user("example@example.com")
+        )
+        return self.import_export_then_validate()
+
+    @targets_models(ApiApplication)
+    def test_apiApplication(self):
+        user = self.create_user()
+        ApiApplication.objects.create(
+            owner=user, redirect_uris="http://example.com\nhttp://sub.example.com/path"
+        )
+        return self.import_export_then_validate()
+
+    @targets_models(ApiToken, ApiApplication)
+    def test_apitoken(self):
+        user = self.create_user()
+        app = ApiApplication.objects.create(
+            owner=user, redirect_uris="http://example.com\nhttp://sub.example.com/path"
+        )
+        ApiToken.objects.create(
+            application=app, user=user, token=uuid4().hex + uuid4().hex, expires_at=None
+        )
+        return self.import_export_then_validate()
+
+    @targets_models(ApiKey)
+    def test_apikey(self):
+        user = self.create_user()
+        self.create_organization(owner=user)
+        ApiKey.objects.create(key=uuid4().hex, organization_id=1)
```

Comment:
nit: here and below, rather than writing out the numerical organization id, we should probably use `self.organization.id` or similar.